docker-compose.yml

```diff
@@ -74,6 +74,17 @@ services:
       DEBUG: "true"
       # Uncomment the following line for MAX logins
       # MAX_URL: https://login.test.max.gov/cas/login
+    depends_on:
+      - ombpdf-js-bundler
+
+  ombpdf-js-bundler:
+    image: node:6
+    volumes:
+      - .:/usr/src/app:delegated
+    environment:
+      USE_POLLING: ${USE_POLLING:-false}
+    working_dir: /usr/src/app/api/ombpdf
+    command: .docker/deps_ok_then node_modules/.bin/webpack --watch
```

Comment:
@cmc333333 is this the kind of thing you were thinking of?  It's completely decoupled from the `ui` app and containers, and it's only available during development, since that's the only time the views that use the JS bundles will be exposed...

Reply:
This is better than what I was thinking of! Having this as a dependency is a great idea and should ensure webpack'll keep running.

Reply:
Oh cool. I would like to talk at some point about what you *were* thinking of, because this was literally the *only* thing I could think of 😆